cameo/api/products.py

```diff
@@ -70,7 +73,15 @@ def search(self, query):
         if len(matches) > 0:
             return matches
         else:
-            raise Exception("No compound matches found for query %s" % query)
```

Comment:
Why shouldn't it raise an Exception (not necessarily `Exception` but a specific one) saying that it didn't found anything?